TP Integration Tests GHA: SE_EVENT_BUS_HOST not set, exiting!

<!--
************ STOP!! ************
If this issue identifies a security vulnerability, DO NOT submit it! Instead, contact
the Apache Traffic Control Security Team at security@trafficcontrol.apache.org and follow the
guidelines at https://apache.org/security regarding vulnerability disclosure.

- For *SUPPORT QUESTIONS*, use the #traffic-control channel on the ASF slack (https://s.apache.org/tc-slack-request)
or the Traffic Control Users mailing list (send an email to users-subscribe@trafficcontrol.apache.org to subscribe).
- Before submitting, please **SEARCH GITHUB** for a similar issue or PR
    * https://github.com/apache/trafficcontrol/issues
    * https://github.com/apache/trafficcontrol/pulls
-->

<!-- Do not submit security vulnerabilities or support requests here - see above -->
## This Bug Report affects these Traffic Control components:
<!-- delete all those that don't apply -->
- Automation <!-- Please specify which (GitHub Actions, Docker images, Ansible Roles, etc.) -->
- unknown

## Current behavior:
<!-- Describe how the bug happens -->
The `chromium` service fails to start in the Traffic Portal Integration Tests GitHub Action:
```
2021-10-14 18:41:20,050 INFO success: selenium-node entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
SE_EVENT_BUS_HOST not set, exiting!
2021-10-14 18:41:20,381 INFO exited: selenium-node (exit status 1; not expected)
```

## Expected behavior:
<!-- Describe what the behavior would be without the bug -->
The `chromium` service should start successfully.

## Steps to reproduce:
<!-- If the current behavior is a bug, please provide the *STEPS TO REPRODUCE* and
include the applicable TC version.
-->
Using the environment variables set in `.github/workflows/tp.integration.tests.yml`:
```yaml
<<'DOCKER_COMPOSE' docker-compose -f- up
version: '3.8'
services:
  hub:
    image: selenium/hub
  chromium:
    image: selenium/node-chrome
    environment:
      HUB_HOST: hub
      HUB_PORT: 4444
    volumes:
      - /dev/shm:/dev/shm
DOCKER_COMPOSE
```

However, the TP Integration Tests will fail, even if all of the services are started correctly. Changes:
```diff
diff --git a/.github/workflows/tp.integration.tests.yml b/.github/workflows/tp.integration.tests.yml
index 03caaaf282..5d7cf96a7b 100644
--- a/.github/workflows/tp.integration.tests.yml
+++ b/.github/workflows/tp.integration.tests.yml
@@ -57,13 +57,17 @@ jobs:
       hub:
         image: selenium/hub
         ports:
-          - "4444:4444"
+          - 4442:4442
+          - 4443:4443
+          - 4444:4444
         options: --health-cmd=/opt/bin/check-grid.sh --health-interval=5s --health-timeout=15s --health-retries=5
       chromium:
         image: ossalternatives/selenium-node-chromium
         env:
-          HUB_HOST: hub
-          HUB_PORT: 4444
+          SE_EVENT_BUS_HOST: hub
+          SE_EVENT_BUS_PUBLISH_PORT: 4442
+          SE_EVENT_BUS_SUBSCRIBE_PORT: 4443
+          SE_NODE_GRID_URL: http://localhost:4444
         volumes:
           - /dev/shm:/dev/shm
       postgres:
```

Log of the end of the *Run TP* step with the above change committed:

```
> trafficportal@1.0.1 build /home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration
> tsc

[19:42:40] I/launcher - Running 1 instances of WebDriver
[19:42:40] I/hosted - Using the selenium server at http://localhost:4444/wd/hub
[19:42:46] E/launcher - Unknown error: {"additionalInformation":"Driver info: driver.version: unknown","localizedMessage":"Could not start a new session. Could not start a new session. Error while creating session with the driver service. Stopping driver service: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=90.0.4430.212 (e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}),platform=Linux 5.8.0-1042-azure x86_64) (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 117 milliseconds\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: 'f38215bb26d8', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown","systemInformation":"System info: host: 'f38215bb26d8', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'","supportUrl":null,"cause":null,"suppressed":[],"stackTrace":[{"fileName":"RemoteNode.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"newSession","className":"org.openqa.selenium.grid.node.remote.RemoteNode","lineNumber":149,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"startSession","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor","lineNumber":513,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"newSession","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor","lineNumber":440,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"handleNewSessionRequest","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable","lineNumber":648,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"lambda$run$1","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable","lineNumber":612,"classLoaderName":"app"},{"fileName":"ThreadPoolExecutor.java","moduleVersion":"11.0.11","moduleName":"java.base","nativeMethod":false,"methodName":"runWorker","className":"java.util.concurrent.ThreadPoolExecutor","lineNumber":1128,"classLoaderName":null},{"fileName":"ThreadPoolExecutor.java","moduleVersion":"11.0.11","moduleName":"java.base","nativeMethod":false,"methodName":"run","className":"java.util.concurrent.ThreadPoolExecutor$Worker","lineNumber":628,"classLoaderName":null},{"fileName":"Thread.java","moduleVersion":"11.0.11","moduleName":"java.base","nativeMethod":false,"methodName":"run","className":"java.lang.Thread","lineNumber":829,"classLoaderName":null}],"message":"Could not start a new session. Could not start a new session. Error while creating session with the driver service. Stopping driver service: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=90.0.4430.212 (e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}),platform=Linux 5.8.0-1042-azure x86_64) (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 117 milliseconds\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: 'f38215bb26d8', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown","rawMessage":"Could not start a new session. Could not start a new session. Error while creating session with the driver service. Stopping driver service: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=90.0.4430.212 (e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}),platform=Linux 5.8.0-1042-azure x86_64) (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 117 milliseconds\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown","buildInformation":{"buildRevision":"3a21814679","releaseLabel":"4.0.0"}}
[19:42:46] E/launcher - WebDriverError: Unknown error: {"additionalInformation":"Driver info: driver.version: unknown","localizedMessage":"Could not start a new session. Could not start a new session. Error while creating session with the driver service. Stopping driver service: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=90.0.4430.212 (e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}),platform=Linux 5.8.0-1042-azure x86_64) (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 117 milliseconds\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: 'f38215bb26d8', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown","systemInformation":"System info: host: 'f38215bb26d8', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'","supportUrl":null,"cause":null,"suppressed":[],"stackTrace":[{"fileName":"RemoteNode.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"newSession","className":"org.openqa.selenium.grid.node.remote.RemoteNode","lineNumber":149,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"startSession","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor","lineNumber":513,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"newSession","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor","lineNumber":440,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"handleNewSessionRequest","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable","lineNumber":648,"classLoaderName":"app"},{"fileName":"LocalDistributor.java","moduleVersion":null,"moduleName":null,"nativeMethod":false,"methodName":"lambda$run$1","className":"org.openqa.selenium.grid.distributor.local.LocalDistributor$NewSessionRunnable","lineNumber":612,"classLoaderName":"app"},{"fileName":"ThreadPoolExecutor.java","moduleVersion":"11.0.11","moduleName":"java.base","nativeMethod":false,"methodName":"runWorker","className":"java.util.concurrent.ThreadPoolExecutor","lineNumber":1128,"classLoaderName":null},{"fileName":"ThreadPoolExecutor.java","moduleVersion":"11.0.11","moduleName":"java.base","nativeMethod":false,"methodName":"run","className":"java.util.concurrent.ThreadPoolExecutor$Worker","lineNumber":628,"classLoaderName":null},{"fileName":"Thread.java","moduleVersion":"11.0.11","moduleName":"java.base","nativeMethod":false,"methodName":"run","className":"java.lang.Thread","lineNumber":829,"classLoaderName":null}],"message":"Could not start a new session. Could not start a new session. Error while creating session with the driver service. Stopping driver service: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=90.0.4430.212 (e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}),platform=Linux 5.8.0-1042-azure x86_64) (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 117 milliseconds\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: 'f38215bb26d8', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown","rawMessage":"Could not start a new session. Could not start a new session. Error while creating session with the driver service. Stopping driver service: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/chromium is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n  (Driver info: chromedriver=90.0.4430.212 (e3cd97fc771b893b7fd1879196d1215b622c2bed-refs/branch-heads/4430@{#1429}),platform=Linux 5.8.0-1042-azure x86_64) (WARNING: The server did not provide any stacktrace information)\nCommand duration or timeout: 117 milliseconds\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown\nBuild info: version: '4.0.0', revision: '3a21814679'\nSystem info: host: '05d554c19192', ip: '172.18.0.3', os.name: 'Linux', os.arch: 'amd64', os.version: '5.8.0-1042-azure', java.version: '11.0.11'\nDriver info: driver.version: unknown","buildInformation":{"buildRevision":"3a21814679","releaseLabel":"4.0.0"}}
npm ERR! Test failed.  See above for more details.
Error on line 156 of /home/runner/work/trafficcontrol/trafficcontrol/.github/actions/tp-integration-tests/entrypoint.sh
    at Object.throwDecodedError (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:520:9)
    at parseHttpResponse (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:519:13)
    at /home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
From: Task: WebDriver.createSession()
    at Function.createSession (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:769:24)
    at Function.createSession (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/chrome.js:761:15)
    at createDriver (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/index.js:170:33)
    at Builder.build (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/selenium-webdriver/index.js:626:16)
    at Hosted.getNewDriver (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/built/driverProviders/driverProvider.js:53:33)
    at Runner.createBrowser (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/built/runner.js:195:43)
    at /home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/built/runner.js:339:29
    at _fulfilled (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/q/q.js:834:54)
    at /home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/q/q.js:863:30
    at Promise.promise.promiseDispatch (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/q/q.js:796:13)
    at /home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/q/q.js:556:49
    at runSingle (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/q/q.js:137:13)
    at flush (/home/runner/go/src/github.com/apache/trafficcontrol/traffic_portal/test/integration/node_modules/protractor/node_modules/q/q.js:125:13)
    at processTicksAndRejections (internal/process/task_queues.js:79:11)
[19:42:46] E/launcher - Process exited with error code 199
Detailed logs produced info Reports artifact
Child process " /home/runner/work/trafficcontrol/trafficcontrol/.github/actions/tp-integration-tests/entrypoint.sh " exited with status code 1 !
0s
```

--- a/.github/workflows/tp.integration.tests.yml
+++ b/.github/workflows/tp.integration.tests.yml
@@ -57,13 +57,16 @@ jobs:
       hub:
         image: selenium/hub
         ports:
-          - "4444:4444"
+          - 4442:4442
+          - 4443:4443
+          - 4444:4444
         options: --health-cmd=/opt/bin/check-grid.sh --health-interval=5s --health-timeout=15s --health-retries=5
       chromium:
         image: ossalternatives/selenium-node-chromium
         env:
-          HUB_HOST: hub
-          HUB_PORT: 4444
+          SE_EVENT_BUS_HOST: hub
+          SE_EVENT_BUS_PUBLISH_PORT: 4442
+          SE_EVENT_BUS_SUBSCRIBE_PORT: 4443
         volumes:
           - /dev/shm:/dev/shm
       postgres:

